brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
```

Comment:
I added a check some lines up to make this function return an error. That will throw a runtime_error. It's just an if in not the heaviest loop at loop and I haven't noticed any significant difference in performance.
In my opinion this is not a logic_error because it's not the users' fault (as opposed to writing e.g. sqrt(-1)), it could be due to a corrupted file.